src/components/MessageDataContent.tsx

```diff
@@ -7,11 +7,33 @@ import MessageBubbleIcon from '../icons/icon-message-bubble.svg';
 import { FunctionCallData } from '../types';
 import { noop } from '../utils';
 
+const ICON_WIDTH = 16;
+const RIGHT_CONTENT_WIDTH = 92;
+const DATA_ROW_GAP = 8;
+
 const Text = styled.div`
   font-size: 14px;
   font-weight: 500;
   line-height: 20px;
   letter-spacing: -0.1px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient
![image](https://github.com/sendbird/chat-ai-widget/assets/26326015/74144404-36bc-42f5-87f2-7c408ae9e2c9)

Reply:
Is it safe?

Reply:
Ah, it's for `-webkit-line-clamp`